examples/README.md

```diff
@@ -0,0 +1,99 @@
+
+`/examples/<version_number>/<category>/<framework>/<example_path>`
+
+- `<version_number>` - the version of the Handsontable and Handsontable's wrapper - the value provided to the NPM scripts as the last argument.
+- `<category>` - category of the code examples. The main category is `docs` which means: code examples from the documentation.
+- `<framework>` - any supported framework's name (react, angular, vue, js).
+- `<example_path>` - path to the specific code example. A folder name of the code example will identify that code example - it will be used as a URL further. The folder name of the code example should be the same across all the frameworks.
+
+An example path to the code example: `examples/8.1.0/docs/angular/custom-id`
+
+It's worth noting, that the `examples` directory is defined as a `npm workspace`, as well as each of the `<framework>` directories within it. This allows installing shared dependencies for all the framework-specific examples.
+
+### Live on production
+
+All code examples are available online. The base URL for the code examples is https://handsontable.github.io/handsontable/ and after the slash comes the path to the built project. 
+
+URL to the specific project follow the undermentioned convention:
+
+`<version_number>/<category>/<framework>/<example_path>`
+
+An example URL to the live example: https://handsontable.github.io/handsontable/8.1.0/docs/js/settings
+
+For more details see the [Folder structure](#folder-structure) section.
+
+### Preview on CodeSandbox
+
+To preview the arbitary code example on CodeSandbox you must get the link to the project folder in the Gihub repo. For example: if the URL to the project is https://github.com/handsontable/handsontable/tree/master/examples/8.0.0/docs/js/settings you can preview that project on CodeSandbox adding the "box" word right after the "github" and before the ".com". The URL to the sandboxed project would be https://githubbox.com/handsontable/handsontable/tree/master/examples/8.0.0/docs/js/settings.
+
+### Creating new examples
+
+Read the [Templates Guideline](./templates/README.md) for the detailed information about new code examples creation.
+
+### Editing existing examples
+
+Sometimes you want to edit existing code examples that live in the `/examples/<version_number>` directory.
+
+1. Edit code examples within the `/examples/<version_number>` directory.
+2. Commit and push the code examples to the repo.
+3. In Github Actions run a manual workflow with the `<version_number>` input of the Handsontable and Handsontable's wrapper version.
+4. The code examples will be deployed to the Github Pages and will be available under the same URL as they already were.
+
+### Development
+
+You can launch the `http-server` or any other server if you want to test and check code examples live.
+
+To see code examples in action run these commands:
```

Comment:
IMO we can remove this line, should be as simple as run npm command, which is explained below
```suggestion
### Development

To see code examples in action run these commands:
```

Reply:
Done https://github.com/handsontable/handsontable/pull/7588#pullrequestreview-602786111